### PR TITLE
Create URDU.md

### LINKED ISSUE #3497: Add Urdu Translation for Readme file

### PR DESCRIPTION
**Pull request by ShadBalti**
Add Urdu readme translation for our project.
Close #3497 

Screenshot:
Screenshot_20240301-185104

### CODE REVIEW

**Comment by OwlbanGroup** [comment]
Create URDU. md

**Review by ShadBalti — COMMENTED** [review]
Approve PR

**Comment by OwlbanGroup** [comment]
Add Urdu Translation for READme file@Owlbangroup

**Comment by OwlbanGroup** [comment]
Create URDU.md #3585

**Review by malakada — CHANGES_REQUESTED** [review]
Wow, thanks so much for taking the time to provide a translation for Urdu!! 😍

@ShadBalti -- I don't speak/read Urdu, so I have some questions about the content if you can help me out. :) I noticed that you left the commit message in English -- would this be common practice for most developers out of Pakistan/India/elsewhere? If so, is there an explanation in Urdu telling the reader that while Urdu may be their primary language that it's customary to write a commit message in English still? If you don't know, I can reach out to some other devs I know in Pakistan at least and ask their opinion.

This PR looks great, but I'd like to ask that you add a few more commits so that it matches what we're looking for with new translations that make it easier for folks to benefit from your work:

- [ ] add a link to your README translation to `/translations/README.md` (https://github.com/Syknapse/Contribute-To-This-Project/tree/master/translations), placed in alphabetical order (looks like it'll be easy for you as it'll go right after Ukranian!)

If you've never worked with Markdown tables before, here's a helpful guide from Github about it: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables, and if you have questions feel free to ping me on the Discord (https://discord.gg/ZtgtthHYUt) in the `#github-help` channel (I'm malakada on Discord as well!). 😄 

Great work over all though and I'm super excited to approve this after you add a link to it!! 🎉

**Comment by malakada** [comment]
Hey @OwlbanGroup -- I know this has been sitting for a long time before getting some attention. Are you still interested in this? If so, we'd be very happy to help you get this merged in. :)

**Comment by OwlbanGroup** [comment]
Yes I am thank you.

**Comment by malakada** [comment]
@OwlbanGroup let's do it then! 🎉 

Just need to add a link to your translation in `/translations/README.md`. I put some helpful links here: https://github.com/Syknapse/Contribute-To-This-Project/pull/3583#pullrequestreview-1965016620

Do you need help with this?

**Comment by OwlbanGroup** [comment]
Yes I do

**Comment by ShadBalti** [comment]
Now it's ready.

**Comment by ShadBalti** [comment]
#3583 ready

**Comment by ShadBalti** [comment]
Close  #3497

File: translations/README/URDU.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  - [مرحلہ 5: کارڈ ٹیمپلیٹ کا نقل کریں](#step-5-copy-the-card-template)
+  - [مرحلہ 6: اپنے تبدیلیوں کو لاگو کریں](#step-6-apply-your-changes)
+  - [مرحلہ 7: اپنی تبدیلیوں کو کمٹ کریں](#step-7-commit-your-changes)
+  - [مرحلہ 8: اپنی تبدیلیوں کو گیت ہب پر پش کریں](#step-8-push-your-changes-to-github)
+  - [مرحلہ 9: پی آر (Pull Request) دیں](#step-9-submit-a-prpull-request)
+  - [مرحلہ 10: جشن منائیں](#step-10-celebrate)
+- [اگلا اقدامات](#next-steps)
+- [شناخت](#acknowledgements)
+- [لائسنس](#license)
+- [اوپر 100 شراکت دار](#top-100-contributors)
```
Comment (TenTraicion) [inline translations/README/URDU.md:44]: the links should be fixed

File: translations/README/URDU.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+## شراکت
+
+### مرحلہ 1: اس ریپازیٹری کو فارک کریں
+
+سب سے پہلے، اس ریپازیٹری کو اپنے گیٹ ہب اکاؤنٹ پر فارک کریں۔ فارک کرنے کے لئے یہ بٹن استعمال کریں۔
+
+[twit]: https://twitter.com/intent/tweet?text=Check%20out%20Contribute-To-This-Project%20by%20Syknapse%20on%20GitHub%20-%20an%20interactive%20tutorial%20on%20making%20your%20first%20contribution%20to%20an%20open%20source%20project%20on%20GitHub%21%20%23OpenSource%20%23GitHub%20%23FirstContribution%20https%3A%2F%2Fsyknapse.github.io%2FContribute-To-This-Project%2F
+
+[![فارک کریں](https://img.shields.io/github/forks/Syknapse/Contribute-To-This-Project.svg?style=social)][twit]
```
Comment (TenTraicion) [inline translations/README/URDU.md:99]: the twit icons should be on top of the file

**Review by TenTraicion — CHANGES_REQUESTED** [review]
hello @ShadBalti , Thanks for your contribution. sorry for the late reply, here are some concerns about your contribution:
- This file seems incomplete
- there are some URL issue in this file
- a link to this file should be in translation doc
- the image contents are missing
Kindly fix those and let me know, and it will be awesome if you can translate the terminal tutorial too.

**Pull request closed without merging**